src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -163,11 +162,11 @@ override public void StartRenameItem()
 			StartRenameItem("ListViewTextBoxItemName");
 		}
 
-		private void ItemNameTextBox_BeforeTextChanging(TextBox textBox, TextBoxBeforeTextChangingEventArgs args)
+		private async void ItemNameTextBox_BeforeTextChanging(TextBox textBox, TextBoxBeforeTextChangingEventArgs args)
 		{
 			if (IsRenamingItem)
 			{
-				ValidateItemNameInputText(textBox, args, (showError) =>
+				await ValidateItemNameInputText(textBox, args, (showError) =>
```

Comment:
Unnecessary async/await is removed in #12010. Is there a reason why async/await should be used?

Reply:
I think something has changed since then, now that method returns an ` async Task`![Screenshot_20230724-123534.png](https://github.com/files-community/Files/assets/102259289/354a3f04-ece5-4988-a962-d23a0e0d5447)

Reply:
The warning can also be avoided by adding `_ = `. I think either is fine.

Reply:
At the moment it uses ` await `  . Would it be better with ` _ ` ?